cmake/scripts/ROOTConfig.cmake.in

```diff
@@ -102,8 +102,25 @@ list(APPEND CMAKE_MODULE_PATH "${ROOT_CMAKE_DIR}/modules")
 #----------------------------------------------------------------------------
 # Find external packages which were used in ROOT
 include(CMakeFindDependencyMacro)
+
 if (NOT ROOT_builtin_nlohmannjson_FOUND)
-  find_dependency(nlohmann_json @nlohmann_json_VERSION@)
+  message(${ROOT_FIND_COMPONENTS})
```

Comment:
swap lines 107 and 108 to avoid an error when calling message. Or just remove 107.

Reply:
I just forget to remove debug output. Will fix